modules/moderation.py

```diff
@@ -600,8 +600,8 @@ async def _strike_set(self, ctx, member: discord.Member, count: StrikeRange, *,
             except discord.Forbidden:
                 if not await tools.mod_cmd_invoke_delete(ctx.channel):
                     await ctx.send(
-                        f'{config.greenTick} {activeStrikes - count} strikes for {member} ({member.id}) have been successfully removed. '
-                        f' They now have {count} strikes now. I was not able to DM them about this action'
+                        f'{config.greenTick} {activeStrikes - count} strikes for {member} ({member.id}) have been successfully removed, '
+                        f' they now have {count} strikes now. I was not able to DM them about this action'
```

Comment:
Make this consistent by removing "now"

Reply:
The "math" should be added to this command too: "they now have X strikes (X - X)"